src/routes/RequestsRoute.js

```diff
@@ -102,8 +102,8 @@ class RequestsRoute extends React.Component {
               'Item barcode': 'item.barcode',
               'Type': 'requestType',
               'Requester': 'requester.lastName requester.firstName',
-              'Requester barcode': 'requester.barcode',
-              'Request date': 'requestDate',
+              'Requester Barcode': 'requester.barcode',
+              'Request Date': 'requestDate',
               'Position': 'position',
               'Proxy': 'proxy',
```

Comment:
TBH, I'm surprised these don't need to match the keys specified in `this.columnLabels` starting on Line 282. Does sorting still work on this branch if your local is something other than `en-US`?